batch.go

```diff
@@ -1,15 +1,18 @@
 package gorgonia
 
+// Batched interface describes any object that can process batch work
 type Batched interface {
 	WorkAvailable() <-chan struct{}
 	DoWork()
 }
 
+// BatchedBLAS interface describes any object that can process BLAS work in batch
 type BatchedBLAS interface {
 	Batched
 	BLAS
 }
 
+// BatchedDevice ...
```

Comment:
I don't know what this interface represents

Reply:
BatchedDevices are the superset of BatchedBlas and the batched CUDA workflow.